src/pykx/pandas_api/pandas_meta.py

```diff
@@ -228,6 +228,14 @@ def max(self, axis=0, skipna=True, numeric_only=False):
             res
         ), cols)
 
+    @convert_result
+    def idxmax(self, axis=0, skipna=True, numeric_only=False):
+        tab = self
```

Comment:
I would remove this line and use `self` directly. It's fine if you want to keep it, since it would be consistent with other functions, just a personal taste.